softlayer/stemcell/softlayer_finder_test.go

```diff
@@ -49,4 +49,26 @@ var _ = Describe("SoftLayerFinder", func() {
 			})
 		})
 	})
+
+	Describe("FindById", func() {
+		Context("valid stemcell ID pointing to a SL virtual disk image", func() {
+			It("returns stemcell and found as true if stemcell ", func() {
+				stemcell, found, err := finder.FindById(4868344)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(found).To(BeTrue())
+
+				expectedStemcell := NewSoftLayerStemcell(4868344, "8c7a8358-d9a9-4e4d-9345-6f637e10ccb7", VirtualDiskImageKind, softLayerClient, logger)
```

Comment:
Moved `expectedStemcell` in `BeforeEach` function.

What do you mean by "I'd check and compare the Id and Uuid of the two objects?" Thanks.